util/cron/test-cray-xc-gpu-arkouda.bash

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#
+# Run arkouda testing on a cray-cs with HDR IB
```

Comment:
Note that file says XC and this says CS.

Reply:
I've updated the comment to:

"# Build Arkouda on a cray-xc with GPU locale model"